### PR TITLE
[ICP-13872] Change power consumption reporting threshold for Qubino 1 Relay

### PR DESCRIPTION
**Pull request by MGoralczykS**
@greens @SmartThingsCommunity/srpol-pe-team 
Could you review this?

### CODE REVIEW

File: devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy [hunk 1]
```diff
@@ -141,6 +141,8 @@ def configure() {
 		// additionally, manual states that default value for below parameters is 0, which disables power reporting
 		cmds += encap(zwave.configurationV1.configurationSet(parameterNumber: 42, size: 2, scaledConfigurationValue: 300))
 		cmds += encap(zwave.configurationV1.configurationSet(parameterNumber: 43, size: 2, scaledConfigurationValue: 300))
+	} else if (zwaveInfo?.model?.equals("0052")) {
+		cmds += encap(zwave.configurationV1.configurationSet(parameterNumber: 40, size: 1, scaledConfigurationValue: 75))
```
Comment (greens) [inline devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy:146]: would we not expect the same behavior from the other devices on this DTH?

Comment (MGoralczykS) [reply devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy:146]: Only two devices in this DTH have power consumption. There are more configuration parameters for Relay 2, but for Relay 1 I needed only to change the value for this parameter.

Comment (greens) [reply devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy:146]: Okay. Can you add a small comment explaining what this is doing here. From what I can gather it changes some threshold for reporting power?

Comment (MGoralczykS) [reply devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy:146]: Fixed. Threshold for low power devices was too small and values were "jumping" all the time

**Review by greens — APPROVED** [review]

**Pull request merged**